### PR TITLE
Adc upgrades

### PR DESCRIPTION
**Pull request by RaulTrombin**
Needs rebase over:
- [x] https://github.com/bluerobotics/navigator-rs/pull/14

Change adc scale according the schematics, using:
 ±4.096 V with LSB - 125 μV
 [FullScale Register](https://www.ti.com/lit/ds/symlink/ads1115.pdf#page=17)

Change the output of adc to a converted value,

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -409,16 +517,35 @@ impl Navigator {
...
+    ///
+    /// let mut nav = Navigator::new();
+    /// nav.init();
+    ///
+    /// loop {
+    ///     println!("ADC channel 1 value: {} [V]", nav.read_adc(AdcChannel::Ch1));
+    ///     sleep(Duration::from_millis(1000));
+    /// }
+    pub fn read_adc(&mut self, channel: AdcChannel) -> f32 {
+        let conversion_volts: f32 = 0.000_125;
```
Comment (joaoantoniocardoso) [inline src/lib.rs:547]: can you explain why this weird value? Might be easier for us to understand if we divide instead of multiply

Comment (RaulTrombin) [reply src/lib.rs:547]: it's the LSB value for ±4.096 V Scale.
It's using 4.096, but it's limited to max 3.3V which is IC's Vdd.

[FullScale Register](https://www.ti.com/lit/ds/symlink/ads1115.pdf#page=17)

**Review by patrickelectric — CHANGES_REQUESTED** [review]
needs rebase

File: src/lib.rs (hunk 1 above)
Comment (patrickelectric) [reply src/lib.rs:547]: can you put the math explicitly on the code ? Or add a comment to make it clear ?
Right now is not possible to understand just looking at the code.

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (RaulTrombin) [reply src/lib.rs]: This is explained with a single comment now. 

I am planning to create a PR to the ads crate. I am considering adding a new method called read_v, which will takes the current scale and perform the conversion.

File: src/lib.rs [hunk 2]
```diff
@@ -94,7 +94,7 @@ pub struct AxisData {
 
 #[derive(Debug)]
 pub struct ADCData {
-    pub channel: [i16; 4],
+    pub channel: [f32; 4],
```
Comment (patrickelectric) [inline src/lib.rs:97]: from the code, it's not clear why it moved to float, can you explain it better ?

Comment (RaulTrombin) [reply src/lib.rs:97]: The PR intends to improve adc methods, we change the raw value to a converted value, thats uses uV per LSB.
Think it's better to have the value converted, like we are doing for all the other sensors.

**Review by joaoantoniocardoso — APPROVED** [review]

**Comment by joaoantoniocardoso** [comment]
@patrickelectric merge at will

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**